src/sentry/auth/superuser.py

```diff
@@ -336,6 +336,11 @@ def enable_and_log_superuser_access():
                 current_datetime=current_datetime,
             )
 
+            metrics.incr(
+                "superuser.success",
```

Comment:
Should this be success or should this be `attempt`?

Reply:
if its reached this point, superuser is already given so i'd say it was successful